### PR TITLE
[bn] Content/en/docs/concepts/windows/ index.md 

### PR DESCRIPTION
**Pull request by sajjadrahman56**
_**localize Content/en/docs/concepts/windows/ index.md  to Bengali**_

Fixes Issue : https://github.com/kubernetes/website/issues/44993
Related PR : https://github.com/kubernetes/website/pull/45009
Tracking Issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

File: content/bn/docs/concepts/windows/_index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+কুবারনেটিস ওয়ার্কার {{< glossary_tooltip text="নোড" term_id="node" >}} লিনাক্স বা মাইক্রোসফ্ট
+উইন্ডোজ চালাতে সহায়তা করে।
+
+{{% thirdparty-content single="true" %}}
+
+CNCF এবং এর মূল লিনাক্স ফাউন্ডেশন সামঞ্জস্যের প্রতি বিক্রেতা-নিরপেক্ষ
+পদ্ধতি গ্রহণ করে। আপনার [উইন্ডোজ সার্ভার](https://www.microsoft.com/en-us/windows-server) এ একটি কুবারনেটিস ক্লাস্টারে
+একটি কর্মী নোড হিসাবে যোগদান করা সম্ভব।
```
Comment (asem-hamid) [inline content/bn/docs/concepts/windows/_index.md:16]: ওয়ার্কার নোড

File: content/bn/docs/concepts/windows/_index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+আপনি [উইন্ডোজে kubectl ইনস্টল এবং সেট আপ ](/bn/docs/tasks/tools/install-kubectl-windows/) করতে পারেন
+আপনার ক্লাস্টারের মধ্যে যে কোন অপারেটিং সিস্টেম ব্যবহার করেন না কেন।
+
+আপনি যদি উইন্ডোজে নোড ব্যবহার করেন তবে আপনি পড়তে পারেন:
+
+* [উইন্ডোজে নেটওয়ার্কিং](/bn/docs/concepts/services-networking/windows-networking/)
+* [কুবারনেটসে উইন্ডোজ স্টোরেজ](/bn/docs/concepts/storage/windows-storage/)
```
Comment (asem-hamid) [inline content/bn/docs/concepts/windows/_index.md:24]: কুবারনেটিসে

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**